nixos/modules/services/networking/stunnel.nix

```diff
@@ -195,7 +195,7 @@ in
                connect = ${v.connect}
                verifyChain = ${yesNo v.verifyChain}
                verifyPeer = ${yesNo v.verifyPeer}
-               ${optionalString (v.CAPath != null) "CApath = ${v.CAPath}"}
+               ${optionalString (v.CAPath != null) "CAfile = ${v.CAPath}"}
```

Comment:
I think there is nothing wrong with this option per se, but just the default value that gets substituted. Maybe it's sufficient to change `${pkgs.cacert}/etc/ssl/certs/ca-bundle.crt` to `${pkgs.cacert}/etc/ssl/certs`